src/main/java/org/folio/rest/impl/BankingInformationAPI.java

```diff
@@ -31,6 +32,7 @@ public BankingInformationAPI() {
   }
 
   @Override
+  @Validate
```

Comment:
It is for case that you mentioned on standup to enable uuil validation?

Reply:
I think `@Validate` is necessary to enable schema validation (not just for uuid). It looks like we just forgot to add these when creating these APIs, so I am adding them in this PR. But it does not work for UUID arrays yet (it should when [RMB-1024](https://folio-org.atlassian.net/browse/RMB-1024) is fixed - although it is possible we would have to change the schema again after the fix).